### DISCUSSION TITLE
Can you release a windows exe installer since the APPX release is unsigned

### DISCUSSION THREAD

**Discussion by dflannery-isw** [mifi/lossless-cut#1551] [Ideas]
Since the APPX release is unsigned it is not useable, as Windows refuses to invoke it.

The reason a regular user would choose the APPX over the portable file dump in the 7z includes simple install, creation of icons, and the capability to uninstall. The runtime environment is of little to no consideration.

Since the provided APPX can't be run because it is unsigned, can you remove it and provide an exe installer instead. Or is it just me??

Thanks for considering. I appreciate your efforts.

**Comment by dflannery-isw** [comment]
Should this be an issue, rather than a question? I didn't want to jump the gun in case it's just me. 
Anyone else have the ability to try the APPX on W10? 
(I'm at 22H2 + KB5023696)

**Comment by dflannery-isw** [comment]
I went back and got an old version LosslessCut-win-x64-3.45.0.exe and my bad, it's not an installer, it's a single file executable. I'd still prefer something like that to the APPX I can't run. Cheers! :-)

**Comment by mifi** [comment]
appx is unsigned and only used for Microsoft Store submissions. I think an installer is not necessary for losslesscut because it is a self contained app that doesnt need to be installed. Just extract 7z and double click
